spec/customer_repository_spec.rb

```diff
@@ -0,0 +1,222 @@
+require 'rspec'
+require 'csv'
+require './data/mockable'
+require './lib/file_io'
+require './lib/customer_repository'
```

Comment:
if we continue to organize alphabetically, we'll have to switch these two at some point.